### DISCUSSION TITLE
张桂梅及华坪女高研究

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#35] [Announcements]
# 张桂梅及华坪女高研究

张桂梅是「七一勋章」的获得者，在官方宣传口径中是完全正面宣传的，但互联网上也存在一些说得有鼻子有眼的负面评价。我很好奇，这些负面评价究竟是否属实，于是有了这次小小的研究。

https://zexwoo.blog/posts/research/zhang-guimei/

**Comment by zhansan379** [comment]
很客观了，牛

**Reply by zhansan379** [reply]
不过农村收入估计会比网上数据低，反正我们村统计的时候，要是写的工资少，村委会还让你多写点。农村都是有活才工作的，很难统计

**Reply by EeeMt** [reply]
不一定吧, 刚上任来管扶贫的统计会想往低了统计, 向上汇报扶贫成果的时候会想往高了统计. 看情况, 灵活的

**Comment by lafaeier** [comment]
佬

**Comment by barrywell** [comment]
有个问题是，张桂梅是靠自己筹款建校的还是靠政府拨款的，如果不是女初中生升学率明显低于男的地方用政府拨款建女高明显是不合理的，建女高这事合理性有待商榷

**Reply by 1050348450** [reply]
你看前面写的啊，靠的政府拨款，性质上是公立学校

**Reply by arkoi111** [reply]
所以说公立学校办单一性别学校有歧视，毕竟男生也有因为家庭困难而辍学的

**Comment by xuenailao** [comment]
我也是看了那个评论感觉疑惑，这里特地留个评论，分析得很有条理！希望有人能进一步实地调查

**Comment by GrayDust404** [comment]
关于入学率，外人难以得知华坪当地的分性别入学率那当然只能从全省的数据看。但是当地政府做决策时理应有相应的决策依据，就连引用[9]那样想用数据说话的文章都只能拿出个辍学原因统计，很难不怀疑真实数据到底是个啥情况啊。

**Comment by twotoinf** [comment]
送你一点数据，反映了近年丽江若干所高中的分性别入学人数情况。
有人说张桂梅的华坪女高挤占了当地男生的机会，真是这样吗？ - solitude rain的回答 - 知乎
https://www.zhihu.com/question/494682758/answer/4684432756

**Reply by ZexWoo** [reply]
感谢提供数据，依托新数据的新分析已经更新上去了。

**Reply by sunsunyuan** [reply]
政府公开信息中女高和一中对比贫困业务经费，似乎女高对贫困户并没有突出贡献。
B站评论区转载：
https://www.bilibili.com/video/BV1PXBNY4EAU/?share_source=copy_web&vd_source=f87a930b50441cec876ab384c20b5771
政府公开信息：丽江女子高中决算
http://www.huaping.gov.cn//xljhpx/c101714/202410/9a83381b9bde44cc8a9b17398ec85a71.shtml

**Comment by arkoi111** [comment]
从高中女童入学率高但毕业率低就知道云南的贫困与歧视

**Reply by arkoi111** [reply]
现在也是有了新认知，她确实是相对对对男生不公，毕竟只招收女生，占用了一些招生计划。忽视了男生因为家庭困难而辍学的情况，作为一所公办学校，我认为是有失的。有缺点的战士终究是战士，至少她在山区教二十年。但是这改变不了他的错误，包括家庭主妇的言论，简直是一派胡言。

**Comment by EeeMt** [comment]
膜拜一下

**Comment by twotoinf** [comment]
招生性别公平性部分的论证太武断了
以华坪二中（初中）2020年为例，其中考上线人数中性别比是男：女=100：109，而全华坪三所公立高中入学性别比是100：140，即使华坪女高按100算依然有100：126
在以上数据分析，且华坪女高分数线未知的基础上，存在两种可能的推断：
1. 华坪女高分数线较低，从而增加了总入学学生中的女生比例
2. 部分男生达到了分数线，但因未知原因不能入学
第一种推断佐证了女高对性别比例的影响，第二种推断与猜想“男生砸锅卖铁上学，女生被迫放弃学业”相悖。

当然这仅仅是基于一所中学中考数据的推断，但一般不应认为不同学校中考上线与性别存在相关关系。
https://mp.weixin.qq.com/s/P3Lq0GYj32KU4GT9yyfUHQ

**Reply by twotoinf** [reply]
华坪二中性别统计来自于知乎，本人没有自行统计过https://www.zhihu.com/question/448371027/answer/1772398173

**Reply by liudidi** [reply]
华坪女高不只招华坪的学生，这部分人要去除。女高确实招部分线下学生，对入学学生的女生比例会有增加。
正常情况下，达线学生都可以上高中（至少我这边是这样，考虑到华坪学校较少，更应该是这样），不上高中的有可能是选择上职业学校，这种情况男生多。另外，华坪的学生也可以上丽江一中等市直高中，一般是尖子生才去，尖子生男生多，所以也可能进一步减少在华坪的男生名额。
总之，如果划统一的录取分数线，女高只招女生和男女同招，对总的录取是没有影响的。如果没有统一的分数线，必然会造成一定影响，但是如果认为女高是增量（也就是没有女高，就没有这些招生名额），其实也是对男生是利好的。

**Comment by cheniiii** [comment]
关于首届民族比例全部是少民的消息来自央视网第二届道德模范评选的介绍，也许后续进一步验证时发现了并不是『全部』都是少民，但这样考虑的话这个绝大多数可能真的是『绝大』
https://museum.cctv.com/special/dierjie/20090617/104942.shtml

**Comment by BurJune** [comment]
性别比不是招生公平性的指标，分数线才是。
按照你的这句话的逻辑，就该支持这个观点：“张就该办男女都招收的高中，而不是女校，因为分数线招生最公平。”
（打个比方，河南和北京上985211的比例不一样，北京的多，河南的少。所以河南靠自己省公共财政支出建了个，只有河南人自己能上的985211，如果你在北京，你考虑一下这个叫教育公平吗。反过来也一样，就得是按你的观点，看分数线最公平，全国一张卷子最公平）

在张这个问题上最大的争议点就是怎么定义什么是教育公平，是分数线还是性别。

**Comment by liudidi** [comment]
张桂梅打算招外国学生的想法只是想法，没有成型，猜测她并不是说要帮助外国学生，而是想扩大学校影响力。2025年女高放在了丽江市第一批招生（以前只有丽江一中），其实就是学校影响扩大的结果。

**Comment by ADMEBC** [comment]
第一：张桂梅从一开始本身是妥妥的县城体制内婆罗门，已故的丈夫是喜洲子弟学校一把手校长，自己的第一份工作就分配到丈夫担任一把手的学校当老师，个中玄妙自行体会。第二：所谓「华坪女高实际上并没有令分数线向着女生倾斜」，简直匪夷所思。女校的存在，相当于是一所男生分数线正无穷、女生分数线略高于华坪一中的歧视性精英学校，凭空（实际上资源软约束下并不是凭空）增加了女性专用的优质学位数量，和公平录取的情况比男生一定录的比女生少。举一个最简单的例子：假如女高公平录取男女，男生一个都达不到华坪女高的录取线？只要男生考上一个，那么就有一个女生要落榜。反过来，因为不公平的男生+∞分数线，就肯定有本来不该占用优质高中非义务教育资源的女生占用了本来属于别人的资源，这就是欺世盗名的不平等。

**Comment by llngji** [comment]
我同意文章大部分分析，但有个地方逻辑明显站不住脚：作为“公办”，女高的名额本该普惠男女，现在却直接限定给女生，这本质上就是牺牲了男性学生的受教育机会。社会层面上山区女孩确实更受压制，但这不等于要用教育制度去反向制造性别不公。所谓的“高分女生去多名女高”，只是因为名额稀缺推高了分数线，而不是教学水平的必然结果。而名额稀缺这件事是因为作为公办学校只招收数量很少的学生.当然这只是一个想象中的推断,:如果没有这所女校而是多一个正常的当地高中呢,招收的女生人数会多,男生人数也会多.把资源倾斜合理化成“优质教育”——这才是最大的问题。

**Comment by Darcy-Zhang** [comment]
谢谢作者！我一直觉得张桂梅本身是伟大的，但是对于张桂梅的宣传是有问题的。谢谢作者做的核对工作

**Comment by CedricYJT** [comment]
其实我觉得你很客观中肯了，但是你没有去思考他的底层逻辑。
你既然已经考虑到他如果是一个仅掐尖录取的普通女高(不存在对贫困家庭的特殊照顾)，那么它不会改变读高中学生的性别比例。
那么你换个角度很快也能意识到，如果当地真的存在远超男性的女孩因贫困上不了高中，那么他只要做到只招贫困生，用任何与性别没有相关性的招生要求都能自然做到大幅度减小女生失学比例。因此，她办女校的初衷就根本站不住脚。
同样，去掐尖录取本身也背离了他办校的初衷。她要去拯救的是失学女高中生，而非学习好的女高中生。如果真的当地扶贫政策非常好，没有多少贫困生上不了学了，那这就会和他当今分数线越卷越高、接近当地高中尖子班平均水平所矛盾。

**Reply by CedricYJT** [reply]
在我看来，这所学校的许多行为都是矛盾的。口号是拯救贫困女生，结果招生分数那么高。就单单一个500人的高中，都已经在当地及周边地区招不到其他分数更低、家庭贫困想上女高的女生了吗？
如果是这样，那么这所学校仍然坚持作为一个女高存在的意义是什么？

**Reply by a550965176-sudo** [reply]
作为一个主要资金来源仍然是财政资金而非社会or个人资金的学校，设置性别门槛本身就挺幽默的。

**Comment by mxkfemkkk** [comment]
道理我都懂，但是“ 华坪女生高中入学率确实高于男生” 一部分的出生性别比波动幅度太大了（一会114一会101一会97的）

短期内可能出现那么大的变化幅度吗？

**Comment by lvr0** [comment]
本文的观点一：
现实情况也确实有可能是总体形势一片大好，而华坪的男女受教育情况「开倒车」，那么在这样的情况下有人振臂一呼说我要帮助女孩儿读书，也是理固宜然的事情。在办校的那个阶段，也确实有一些数据可以从侧面印证。
本文观点二：
这一部分数据能说明的问题是，宁蒗县在经济条件落后的情况下，招生情况优于华坪，而且华坪总体经济在丽江市内排第二，说明总体上经济没有成为导致娃儿念不上书的原因，因贫穷而失学应是比较少的情况。

这两个观点有点矛盾而且我认为观点二数据的证明力强于观点一的。
观点一的数据来源调查了云南全省农村入学率的情况以及包括云南在内其他省份的男女性别以及辍学的原因。
调查内容显示其辍学原因的因为经济原因辍学的女性高于男性，但是该内容并非云南甚至华坪县当地的情况。
云南确实存在早婚早育的排名靠前的情况，但观点二数据既然可以得出华坪女性总体并没有因经济原因辍学，那观点一这组数据来源在华坪范围就需要重新调查并证明早婚早育数量和女性高中入学率正相关。
观点二数据虽然也并非华坪本县，但同样在云南境内，可靠程度高于观点一。

华坪女高主要资金来源是财政拨款，也就是公共资源，哪怕是专项拨款也是来源全国人民的资金，项目必要性应该进行充分论证，起码应该由负责拨款的人或者项目负责人出一份调查报告。

最后，关于有缺陷的战士这部分结论，哪怕作者文章相当客观，数据相当翔实，这个结论还是值得商榷的。
并不能证明把专项财政资金交由张桂梅分配而不是教育局分配更能兼具效率和公平，如果更效率和公平应该可以惠及更多的贫困的学生。哪怕她本人可能是没有什么坏心，但是从专门招收女性以及招收外国人看来，其决策和行动可能是盲动的，可能破坏了原本公平和效率的决策。

**Comment by extrazzb** [comment]
为什么引用中的华坪县相关预算链接(以huaping.gov.cn为前缀)均不能打开了? 是领导班子塌方后全面屏蔽了吗
